report_sources/elr_review.Rmd

```diff
@@ -586,12 +597,17 @@ out_final <- out_full %>%
     growth_rate = r,
     relative_incidence = perc_peak,
     net_increases = net_increases,
-    last_week_tpr = tpr,
-    tpr_change = change_tpr,
+    # remove old tpr for now
+    # last_week_tpr = tpr,
+    # tpr_change = change_tpr,
     epi_classification = classif,
     growth_rate_lower = r_lower,
-    growth_rate_upper = r_upper,
-    growth_rate_tpr = r_tpr)
+    growth_rate_upper = r_upper
+    # growth_rate_tpr = r_tpr
+  )
+
+out_final <- out_final %>% 
+     left_join(elr_extras_final, by = "iso3")
```

Comment:
It looks like this is the first time 'elr_extras_final' is defined, which causes the following error on my end when compiling the Rmd:
```
label: unnamed-chunk-18
Quitting from lines 572-612 (elr_review.Rmd) 
Error in is.data.frame(y) : object 'elr_extras_final' not found
```